.travis.yml

```diff
@@ -89,7 +89,7 @@ jobs:
   - env: SQLFLOW_TEST=workflow
     script:
     - set -e
-    - bash scripts/test/setup_k8s_env.sh
+    - $TRAVIS_BUILD_DIR/scripts/travis/setup_minikube.sh
```

Comment:
I added `chmod +x` to almost all of our shell scripts and removed the `source *.sh` and `bash *.sh` and `sh -c *.sh` invocations.